src/compiler/php_generator.cc

```diff
@@ -18,10 +18,12 @@
 
 #include <map>
 
+#include <google/protobuf/compiler/php/php_generator.h>
 #include "src/compiler/config.h"
 #include "src/compiler/generator_helpers.h"
 #include "src/compiler/php_generator_helpers.h"
 
+using google::protobuf::compiler::php::GeneratedClassName;
```

Comment:
Hi, @nicolasnoble . Can you please give me an advise here? Do I need to move include `#include <google/protobuf/compiler/php/php_generator.h>` to config.h and make it `grpc::protobuf::compiler::php`?